src/Psl/DateTime/DateTimeConvenienceMethodsTrait.php

```diff
@@ -616,10 +616,9 @@ public function toString(null|DateStyle $date_style = null, null|TimeStyle $time
         $timestamp = $this->getTimestamp();
 
         /**
-         * @psalm-suppress InvalidOperand
          * @psalm-suppress ImpureMethodCall
          */
         return Internal\create_intl_date_formatter($date_style, $time_style, null, $timezone ?? $this->getTimezone(), $locale)
-            ->format($timestamp->getSeconds() + ($timestamp->getNanoseconds() / NANOSECONDS_PER_SECOND));
+            ->format($timestamp->getSeconds());
```

Comment:
the default pattern does not represent nanoseconds.
it is not possible (to my knowledge) to get a timestamp to contain nanoseconds > NANOSECONDS_PER_SECOND - 1. 

I did not find a case to cover this calculation